app/main.py

```diff
@@ -2,0 +1,14 @@
+def copy_file(command: str) -> None:
+    command = command.split()
```

Comment:
Code Style Issue: It's better to use a more descriptive variable name instead of `command`. You could name it `command_parts` or `command_tokens` to make it clear that it's a list of the parts of the command.